src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -1,6 +1,25 @@
 <template>
   <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="add-notation-dialog">
-    <v-card>
+    <v-card v-if="name==='conversion'">
```

Comment:
Use FilingTypes enum instead of hard-coded string.

For this to work, you will need to add:
```
// declaration for template
readonly FilingTypes = FilingTypes
```

Reply:
I removed this modal.